src/broker/exec.c

```diff
@@ -30,34 +30,34 @@
 
 static void exec_finalize (void *arg)
 {
-    flux_subprocess_server_t *s = arg;
-    flux_subprocess_server_stop (s);
+    subprocess_server_t *s = arg;
+    subprocess_server_stop (s);
 }
 
 void exec_terminate_subprocesses (flux_t *h)
 {
-    flux_subprocess_server_t *s = flux_aux_get (h, "flux::exec");
+    subprocess_server_t *s = flux_aux_get (h, "flux::exec");
 
     /* exec_initialize() never called */
     if (!s)
         return;
 
-    if (flux_subprocess_server_subprocesses_kill (s,
+    if (subprocess_server_subprocesses_kill (s,
                                                   SIGTERM,
                                                   EXEC_TERMINATE_TIMEOUT) < 0)
-        flux_log_error (h, "flux_subprocess_server_subprocesses_kill");
+        flux_log_error (h, "subprocess_server_subprocesses_kill");
 
     /* SIGKILL is also sent in final teardown via
-     * flux_subprocess_server_stop(), but we wish to avoid any
+     * subprocess_server_stop(), but we wish to avoid any
      * subprocesses continuing to run and potential send broker
      * messages while we begin teardown, so we will SIGKILL the
      * subprocesses as well.
      */
 
-    if (flux_subprocess_server_subprocesses_kill (s,
+    if (subprocess_server_subprocesses_kill (s,
                                                   SIGKILL,
                                                   EXEC_TERMINATE_TIMEOUT) < 0)
-        flux_log_error (h, "flux_subprocess_server_subprocesses_kill");
+        flux_log_error (h, "subprocess_server_subprocesses_kill");
```

Comment:
nit: in `libsubprocess: drop flux_ prefix from server API` correct tabbing (but appears code will disappear in later commit)